internal/pkg/pluginengine/fillRefValueWithOutputs_test.go

```diff
@@ -0,0 +1,223 @@
+package pluginengine
+
+import (
+	"github.com/merico-dev/stream/pkg/util/log"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/merico-dev/stream/internal/pkg/configloader"
+	"github.com/merico-dev/stream/internal/pkg/statemanager"
+)
+
+// TestOutputsInState normal test, if the ref value is correct
+func TestOutputsInState(t *testing.T) {
+	keyReferrer := statemanager.StateKey("default-trello-github_trello-github-integ")
+	stateReferrer := statemanager.State{
```

Comment:
It seems this part is duplicated code across all test cases. Similar for the defer part. Maybe we can utilize setup/teardown mechanism for this